scalahost/nsc/src/main/scala/scala/meta/internal/semantic/ConfigOps.scala

```diff
@@ -2,17 +2,38 @@ package scala.meta.internal
 package semantic
```

Comment:
I don't mind either way, just wanted to figure out your preferences wrt this aspect of code organization.